{result="_error", invokeId=5, serviceCall=NULL, data={timestamp=1.37442e+012, faultString="com.riotgames.platform.game.GameNotFoundException : No Game for player 366029 was found in the system!", correlationId="3A0E955C-2FD0-2537-5904-C7FA2BFA4199", body=NULL, messageId="43CE3A79-B10E-032A-F17D-8BBB2608A106", destination="gameService", rootCause={substitutionArguments=["366029"], errorCode="PG-0014", rootCauseClassname="com.riotgames.platform.game.GameNotFoundException", localizedMessage="No Game for player 366029 was found in the system!", cause=NULL, message="No Game for player 366029 was found in the system!"}, timeToLive=0, clientId="43CE3A6D-7C38-B3CD-C9F9-7097D97CE7FA", faultCode="Server.Processing", headers={}, extendedData=NULL, faultDetail=NULL}}